autometacal/python/datasets/galaxies.py

```diff
@@ -8,214 +8,181 @@
 Functions with gs_ prefix depend on galsim
 """
 
+def simple(
+  snr = 100.,
+  gal_hlr = 0.7,
+  gal_g1 = 0.,
+  gal_g2 = 0.,
+  psf_g1 = 0.,
+  psf_g2 = 0.,
+  flux = 1.,
+  **kwargs
+):
+  """Simple exponential profile toy model galaxy"""
 
-def gs_generate_images(**kwargs):
-  
-  """ Random Galaxy Generator
-  Generates noiseless galaxy images with a simple light profile. The resulting image is before the convolution with a PSF. 
-  Galaxy shapes follow a bivariate normal distribution centered in zero.
-  
-  Args:
-    g_range: galaxy shapes go from -g_range and + g_range in each g1, g2 
-    g_scatter: galaxy shapes scatter
-    flux: galaxy flux (counts)
-    pixel_scale: intended pixel scale in arcsec^2/pixel
-    stamp_size: size in pixels of the NxN resulting image
-    method: galsim drawing method
-    mean_radius: mean half-light radii of generated galaxies
-    scatter_radius: scatter of half-light radii of generated galaxies
-    mean_snr: average snr of generated stamps (snr as per galsim definition)
-    scatter_snr: scatter in snr of generated stamps
-    interp_factor: interpolation factor for drawing k space images
-    padding_factor: padding factor for drawing k space images
-  Returns:
-    g1, g2: galaxy shape parameters
-    gal: tensor with a 2-d array representing an observed image of a galaxy (with convolved psf)
-    psf: tensor with a 2-d array representing the model of the psf
-    gal_k: k space image of gal
-    psf_k: k space image of psf
-  """
-
-  defaults = {'g_range' : 0.8,        #elipticity
-              'g_scatter' : 0.25,     #
-              'mean_radius': 1.0,     #size
-              'scatter_radius': 0.1,  #
-              'psf_beta': 5,          #psf
-              'psf_fwhm': 0.7,        #
-              'mean_snr': 200,        #snr
-              'scatter_snr': 20,      #
-              'flux' : 1e5,           #flux
-              'pixel_scale' : 0.2,    #
-              'stamp_size' : 50,      #
-              'method' : "no_pixel",   #
-              'interp_factor': 2,     #kimage interpolation
-              'padding_factor': 1     #kimage padding
-             }
-
+  #defaults are those expected to not vary much inside a sample
+  defaults = {
+    'scale' : 0.2,
+    'stamp_size' : 51,
+    'psf_fwhm' : 0.9, 
+  }
```

Comment:
This could be set as a "global" variable since it is used in all of your functions